ivadomed/config/config_vertebral_labeling.json

```diff
@@ -27,7 +27,7 @@
     },
     "split_dataset": {
         "fname_split": null,
-        "random_seed": 8,
+        "random_seed": 6,
```

Comment:
Curious about your thoughts on centralizing the seeds all to a single place. So if that to happen, would we still need to indicate `random_seed` here? Also, just curious why this particular JSON was changed instead of the main `config.json`?

I see below, you used this as the anchor to set the random seed for all dependencies. I agree from a practical perspective, this is the "quickest" way to get to proper seeding with minimal change to the code base. 

From a design/user configuration perspective, do you think it would easier for you (as a user role) to see this setting at the "top" configuration json level instead? You have probably seen how seeding is being done in other libraries and code bases, any interesting examples we can reference here as a good practice example?

Reply:
> Also, just curious why this particular JSON was changed instead of the main config.json?

I don't really understand what you mean here. This is the main `config.json` file, unless I might have changed something that has skipped my mind.

> From a design/user configuration perspective, do you think it would easier for you (as a user role) to see this setting at the "top" configuration json level instead? You have probably seen how seeding is being done in other libraries and code bases, any interesting examples we can reference here as a good practice example?

Most of the libraries/codebases I have come across use the seed at the top level. Initially, that was what I did, but @andreanne-lemay  and @mariehbourget mentioned that would mean changes to the existing config files (Comment is at the top), so I had to undo everything and use the one inside `split_dataset` as the anchor. I don't think it is a good design principle to do that, but it is a fix for now until I can work around the `config_manager` file to make it "retrospective".

Reply:
Good point. Thanks for the discussion. Regarding the first point. I think [config_vertebral_labeling.json] is only a specific config.json instead of the [main config.json](https://github.com/ivadomed/ivadomed/blob/master/ivadomed/config/config.json)? So depending on what your goal is, would it be more applicable to modify the main config JSON file?